src/config/version-types.js

```diff
@@ -0,0 +1,38 @@
+export default [
+  {
+    icon: 'vm-icon',
+    name: 'VM',
+    value: 'vmbased',
+    intro: 'delivery_type_intro_vm'
+  },
+  {
+    icon: 'helm-icon',
+    name: 'Helm',
+    value: 'helm',
+    intro: 'delivery_type_intro_helm'
+  },
+  {
+    icon: 'saas-icon',
+    name: 'SaaS',
+    value: 'saas',
```

Comment:
`SaaS, API, Native, Serveless, ` these types need to add a property `disable: true`.

Reply:
ok